helmfile/charts/opensearch/slm/values.yaml

```diff
@@ -23,6 +23,7 @@ snapshots:
   maxRequestSeconds: 1200
 
 restartPolicy: OnFailure
+backoffLimit: 0
```

Comment:
I don't think this makes sense for opensearch-slm and sc-logs-rentention.
Especially opensearch-slm sometimes needs retries since it relies on OpenSearch to actually carry out the clean up which may take a while depending on the number of snapshots and indices in them.

Reply:
Agreeing with @aarnq

Reply:
Sure i can remove it from slm. But i keep it for opensearch backup.

Reply:
I would still only want the backoff limit set on the rclone jobs